gramps/gui/views/treemodels/treebasemodel.py

```diff
@@ -568,10 +568,10 @@ def _rebuild_filter(self, dfilter, dfilter2, skip):
             # The tree has both primary and secondary data. The navigation type
             # (navtype) which governs the filters that are offered, is for the
             # secondary data.
-                items = self.number_items2()
-                _LOG.debug("rebuild filter secondary")
-                self.__rebuild_filter(dfilter2, skip, items,
-                                      self.gen_cursor2, self.map2, self.add_row2)
+            items = self.number_items2()
+            _LOG.debug("rebuild filter secondary")
+            self.__rebuild_filter(dfilter2, skip, items,
+                                    self.gen_cursor2, self.map2, self.add_row2)
 
```

Comment:
As a PR, I can see the context of this code. Thanks!

You are also fixing these non-related indentation issues. I'm surprised that this wasn't a compile error before.